packages/api/src/controllers/project.ts

```diff
@@ -0,0 +1,163 @@
+import { Request, RequestHandler, Router } from "express";
+import { authorizer, validatePost } from "../middleware";
+import { db } from "../store";
+import { v4 as uuid } from "uuid";
+import {
+  makeNextHREF,
+  parseFilters,
+  parseOrder,
+  getS3PresignedUrl,
+  toStringValues,
+  pathJoin,
+  getObjectStoreS3Config,
+  reqUseReplica,
+  isValidBase64,
+  mapInputCreatorId,
+} from "./helpers";
+import sql from "sql-template-strings";
+import {
+  ForbiddenError,
+  UnprocessableEntityError,
+  NotFoundError,
+  BadRequestError,
+  InternalServerError,
+  UnauthorizedError,
+  NotImplementedError,
+} from "../store/errors";
+
+const app = Router();
+
+export async function getProject(req, projectId) {
+  let project;
+
+  if (!req.user.admin && req.user.id !== project.userId) {
```

Comment:
Is this function used? Looks like it will always get a null reference exception here since `project` is unassigned. Overall, looks like its implementation can be reduced to
```ts
const project = projectId ? await db.project.get(projectId) : { name: 'default', userId: req.user.id }
if (!req.user.admin && req.user.id !== project.userId) {
   throw new ForbiddenError(`invalid user`);
}
```
WDYT?

Reply:
Good point - the function is used and it seemed to work even though project was left unassigned. But I see your point though - updated to use your suggestion as it's simpler and effectively does the same.